README.md

```diff
@@ -1,23 +1,158 @@
 
-const result = await multiply(3, 7);
+```xml
+<manifest ... >
+    ...
+    <application android:allowBackup="true" ... >
+        ...
+    </application>
+</manifest>
 ```
 
+## API
+
+### `put`
+
+Stores a string value in the storage with the given key.
+
+```ts
+try {
+  const key = 'key';
+  const value = 'value';
+  await SecureStorage.put(key, value);
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `get`
+
+Retrieves the value with the given key from the storage. If the key does not exist, the method will throw an error.
+
+```ts
+try {
+  const key = 'key';
+  const value = await SecureStorage.get(key, value);
+  console.log(value); // 'value'
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `remove`
+
+Removes the value with the given key from the storage.
+
+```ts
+try {
+  const key = 'key';
+  await SecureStorage.remove(key, value);
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `clear`
+
+Deletes all keys and values from the storage.
+
+```ts
+try {
+  await SecureStorage.clear();
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `keys`
+
+Returns an array of all keys in the storage.
+
+```ts
+try {
+  const keys = await SecureStorage.keys();
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `setEnforceManualEncryption` (Android Only)
+
+This method enables manual encryption on Android. It should be called before any other method. If the directory holding application files is already encrypted by default, then manual encryption is disabled. Enabling it manually results in a double encrypted file.
+
+```ts
+try {
+  const keys = await SecureStorage.keys();
+} catch (e) {
+  const error = e as SecureStorage.SecureStorageError;
+  setStatus(`Error: ${error.message}`);
+  console.log(JSON.stringify(e));
+}
+```
+
+### `tests` (Android Only)
+
+This method runs a test suite on Android to check if the library is working correctly. [SecuraStorageInstrumentedTest.kt] already contains an instrumented test suite which can be run through Android Studio. However, this method is useful for running tests from the JavaScript side.
```

Comment:
Why `SecuraStorageInstrumentedTest.kt` is inside square brackets and not formatted as code? 🤔

Reply:
Copy paste from Kotlin documentation, fixed in 68bd55e9406425a4d835696462b466b097ac2a39, thanks.